test/system/260-sdnotify.bats

```diff
@@ -332,20 +334,26 @@ ignore"
     run_podman logs $container_a
     is "${lines[0]}" "/run/notify/notify.sock" "NOTIFY_SOCKET is passed to container"
 
-    # Instruct the container to send the READY
+    # Send the READY message.  Doing it in an exec session helps debug
+    # potential issues.
+    run_podman exec --env NOTIFY_SOCKET="/run/notify/notify.sock" $container_a /usr/bin/systemd-notify --ready
```

Comment:
Clever. I'm surprised it actually works.

Reply:
Thanks!  I was quite happy about this change during development.